sleap/gui/color.py

```diff
@@ -177,6 +195,22 @@ def get_track_color(self, track: Union[Track, int]) -> ColorTupleType:
             return (0, 0, 0)
 
         return self.get_color_by_idx(track_idx)
+    
+    def get_instance_group_color(self, instanceGroup: Union[InstanceGroup, int]) -> ColorTupleType: 
+        """Returns the color to use for a given instance group.
+
+        Args:
+            instanceGroup: `InstanceGroup` object
+        Returns:
+            (r, g, b)-tuple
+        """
+        instanceGroup_idx = instanceGroup
+        if isinstance(instanceGroup, InstanceGroup):
+            instanceGroup_idx = self.instanceGroups.index(instanceGroup) if instanceGroup in self.instanceGroups else None
+        if instanceGroup_idx is None:
+            return (0, 0, 0)
+        
+        return self.get_color_by_idx(instanceGroup_idx)
```

Comment:
`snake_case` please!
```suggestion
        instance_group_idx = instance_group
        if isinstance(instance_group, InstanceGroup):
            instance_group_idx = self.instance_groups.index(instance_group) if instance_group in self.instance_groups else None
        if instance_group_idx is None:
            return (0, 0, 0)
        
        return self.get_color_by_idx(instance_group_idx)
```